firefox-ios/Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -104,8 +104,13 @@ class TelemetryWrapper: TelemetryWrapperProtocol, FeatureFlaggable {
             GleanMetrics.LegacyIds.clientId.set(uuid)
         }
 
+        glean.registerPings(GleanMetrics.Pings.shared)
+
         // Initialize Glean telemetry
-        let gleanConfig = Configuration(channel: AppConstants.buildChannel.rawValue, logLevel: .off)
```

Comment:
Is logLevel being deprecated? Will Glean logs show up in the console again or is it just defaulted to off?

Reply:
Woops, thanks for catching that. I removed it for local testing, and then promptly forgot to put it back.